cypress/e2e/functional/case-retention-spec.cy.js

```diff
@@ -130,12 +130,15 @@ describe('Case retention screen as standard user', () => {
       cy.get('.govuk-error-message').should('exist');
       cy.get('.govuk-error-summary').should(
         'contain',
-        'You do not have permission to reduce the current retention date. Please refer to the DARTS retention policy guidance'
+        'You do not have permission to reduce the current retention date'
```

Comment:
Are these changes related to the ticket?

Reply:
Removed and stashed for another branch